internal/k8s/apply.go

```diff
@@ -12,5 +13,9 @@ func Apply(ctx context.Context, rawYAML string) error {
 	r := bytes.NewReader([]byte(rawYAML))
 	c.Stdin = r
 
+	// TODO: handle output more cleanly
+	c.Stdout = os.Stdout
```

Comment:
Do we want this printing to the screen?

Reply:
ya, i'm also not totally clear on what you're trying to accomplish here. You can pull Stderr out of an error if that's what you're looking for